services/base-service/models/grant_role/model.js

```diff
@@ -49,7 +49,7 @@ class GrantRoleModel extends BaseModel {
     let expert;
     let resp;
 
-    logger.info({patch},`grant_role.patch for ${expertId}:`);
+    logger.info(`grant_role.patch for ${expertId}:`,patch);
```

Comment:
# Example 1 # 
Completely unexpected errors can be tested by eg. replacing this line with 
```javascript
throw new Error('Unexpected Error')
```

Reply:
This Error propagates the the Error handler and the Response is of type (500) (since we didn't expect this error) and the message is : 

```html 
<body>
<pre>Error: Unexpected Error<br> &nbsp; &nbsp;at AuthorshipModel.patch (/fin/services/models/authorship/model.js:50:11)<br> &nbsp; &nbsp;at /fin/services/models/expert/api.js:145:32<br> &nbsp; &nbsp;at process.processTicksAndRejections (node:internal/process/task_queues:95:5)</pre>
</body>
```